gio/src/auto/data_input_stream.rs

```diff
@@ -122,8 +124,11 @@ pub trait DataInputStreamExt: 'static {
     #[doc(alias = "g_data_input_stream_read_int64")]
     fn read_int64<P: IsA<Cancellable>>(&self, cancellable: Option<&P>) -> Result<i64, glib::Error>;
 
-    //#[doc(alias = "g_data_input_stream_read_line_finish_utf8")]
-    //fn read_line_finish_utf8(&self, result: /*Ignored*/&AsyncResult) -> Result<(Option<glib::GString>, usize), glib::Error>;
+    #[doc(alias = "g_data_input_stream_read_line_finish_utf8")]
+    fn read_line_finish_utf8<P: IsA<AsyncResult>>(
```

Comment:
Needs a change to gir so that all the `finish()` functions are not generated. They're already handled as part of the `async()` functions

Reply:
Sure, I will add rules for those functions in the Gir

Reply:
I meant in https://github.com/gtk-rs/gir , not in the `Gir.toml` :)